src/lib/proxy/proxy-controller.ts

```diff
@@ -1,8 +1,8 @@
 import { Request, Response } from 'express';
-import Controller from '../controller';
-import { IUnleashConfig, IUnleashServices, NONE } from '../../types';
-import { Logger } from '../../logger';
-import { IApiUser } from '../../types/api-user';
+import Controller from '../routes/controller';
```

Comment:
I'd like the file to be named "frontend-api-controller" to be more consistent with what we refer to it as internally. This should also be the controller name.